tests/sdk/python/python_test.go

```diff
@@ -360,6 +361,34 @@ func TestHelmLocal(t *testing.T) {
 	}
 	options := baseOptions.With(integration.ProgramTestOptions{
 		Dir: filepath.Join(cwd, "helm-local", "step1"),
+		ExtraRuntimeValidation: func(t *testing.T, stackInfo integration.RuntimeValidationStackInfo) {
```

Comment:
This is neat! DId you verify this fails before the fix?

Reply:
Yep! Without the fix, both ConfigMap resources will create before the Deployment and Service.